src/EditorFeatures/CSharpTest/Interactive/CodeActions/InteractiveIntroduceVariableTests.cs

```diff
@@ -213,5 +213,70 @@ void M(params string[] args)
 }
 ");
         }
+
+        [WorkItem(45790, "https://github.com/dotnet/roslyn/issues/45790")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        public Task IntroduceLocal_RValueOnly() =>
```

Comment:
please follow the style of the tests around you.  First, we use blocks here uniformly, so this code is now inconsistent.  Second, our IDE style is that multiline methods that use `=>` place the `=>` on the next line indented 4.

Finally, we do an await in the test code itself so that an exception here shows up nicely in this stack frame and not the higher one calling this.